Emby.Naming/Common/NamingOptions.cs

```diff
@@ -176,7 +177,7 @@ public NamingOptions()
 
             CleanDateTimes = new[]
             {
-                @"(.+[^ _\,\.\(\)\[\]\-])[ _\.\(\)\[\]\-]+(19[0-9][0-9]|20[0-1][0-9])([ _\,\.\(\)\[\]\-][^0-9]|$)"
+                @"(.+[^_\,\.\(\)\[\]\-])[_\.\(\)\[\]\-](19[0-9][0-9]|20[0-1][0-9])([ _\,\.\(\)\[\]\-][^0-9]|).*(19[0-9][0-9]|20[0-1][0-9])*"
```

Comment:
why? this ain't any test! stop sneaking unrelated stuff please :D

Reply:
I think it was required to make some of the tests pass, which is why it was changed.

Reply:
Dkanada is correct, it’s needed for some tests. Got this from the last Emby.Naming release

Reply:
Which tests are failing?

Reply:
```cs
        [Fact]
        public void TestMultiEdition3()
        {
            var files = new[]
            {
                @"/movies/The Phantom of the Opera (1925)/The Phantom of the Opera (1925) - 1925 version.mkv",
                @"/movies/The Phantom of the Opera (1925)/The Phantom of the Opera (1925) - 1929 version.mkv"
            };

            var resolver = GetResolver();

            var result = resolver.Resolve(files.Select(i => new FileSystemMetadata
            {
                IsDirectory = false,
                FullName = i

            }).ToList()).ToList();

            Assert.Single(result);
            Assert.Single(result[0].AlternateVersions);
        }
```
This one

Reply:
Care to explain the changes to the regex then? I can see where their results differ, but why did you e.g. removed whitespace from first group?

Reply:
Like I said, I just copied it from the latest Emby.Naming release